tests/unit/mixins/test_mixin_methods.py

```diff
@@ -35,7 +35,7 @@ class M(GetMixin, FakeManager):
         url=url,
         json={"id": 42, "foo": "bar"},
         status=200,
-        match_querystring=True,
+        match=[responses.matchers.query_param_matcher({})],
```

Comment:
Sure can do that.

My goal in this PR was to try to do as minimal as changes as possible needed to do the conversion. Meaning no optimizations or clean-up.

Another goal of mine s to have a follow-on to this PR to reduce the duplication. But I'll be honest and say I have not yet looked closely to validate that it is my possible. I think it is but not at 100% certainty yet.

Reply:
Ok, follow-up works for me as well! Yeah, I saw there is a lot of repetitive mocking inside the test cases, but I agree that's for later.

Reply:
I did a quick try with `autouse` and it didn't work out how I expected it would. Looking at the docs it seems like `autouse` fixtures will be called on the arguments passed in. Which if so is not what we want.

So I used a global variable instead to make changing easy.